### ISSUE TITLE
GRCh38 coordinates

### ISSUE THREAD

**Issue by gerthijs** [griffithlab/civic-server#429]
Currently coordinates of variants are provided in GCRh37, would it be possible to also return the coordinates in GRCh38?

**Comment by malachig** [comment]
Yeah, there are various ways we could approach this.  We could do the conversion internally.  I think what we should aim for is adopt a standard approach to solving this problem.  There are many conversions that may be of interest. For example,

- Between genome builds (hg19, GCRh37, GRCh38)
- Different transcript sequences or transcript annotation systems (Ensembl, GENCODE, LRG, etc.)
- DNA <-> RNA <-> Protein
- Representation styles (HGVS, VCF, VMC, etc.)

The last thing the world needs is another implementation of these conversions.  Our present goal is to tie everything possible to a ClinGen Allele Registry ID.  This does not support all variant types, but it is a good start.  We will keep our eyes on the development of VMC implementations as well.

Here is an example variant in CIViC that has been linked to Allele Registry:
https://civicdb.org/events/genes/19/summary/variants/33/summary

Here is the Allele Registry page for that variant:
https://reg.genome.network/redmine/projects/registry/genboree_registry/by_caid?caid=CA126713

JSON-LD response
https://reg.genome.network/allele/CA126713

It looks like it solves a lot of the use cases I list above.  Since it would be convenient to many users of CIViC we can pull *some* of this data into CIViC. 

The build38 coordinates should be at the top of the list for things to pull in.

**Comment by gerthijs** [comment]
Is there any progress on providing GRCh38 coordinates for variants? 
We see that more and more labs are switching to GRCh38 kits and designs and having the CIViC data available in GRCh38 would be helpful for these labs.

**Comment by gerthijs** [comment]
Has any decision or progress been made on the use of GRCh38 coordinates in CIViC?
